fail2ban/server/banmanager.py

```diff
@@ -26,6 +26,9 @@
 
 from threading import Lock
 
+import dns.exception
```

Comment:
what about making this all optional, e.g. delay imports till the call, and then if not available -- spitting out informative message and then report old style limited output?